miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -117,7 +119,16 @@ internal open class MiniAppWebView(
                 String.format("%s %s", settings.userAgentString, hostAppUserAgentInfo)
 
         setupMiniAppNavigator()
-        webViewClient = getMiniAppWebViewClient()
+
+        webViewClient = if (requireAdPlacementBeta) {
+            MiniAppH5AdsWebViewClient(context).getH5AdsWebViewClient(
```

Comment:
See my other comment - we should check here whether the class is present so that they aren't required to have the `admob-latest` or Ads SDK dependencies if they don't need them.